app/scripts/directives/createSecret.js

```diff
@@ -70,6 +70,15 @@ angular.module("openshiftConsole")
                 label: "Webhook Secret"
               }
             ]
+          },
+          opaque: {
+            label: "Opaque Secret",
```

Comment:
Let's call it `Generic Secret` to match the CLI command (`oc create secret generic ...`)

Reply:
Bugzilla and Build Inputs documentation seem to use ```oc secrets new secret-npmrc .npmrc=~/.npmrc```

Reply:
```
❯ oc create secret generic -h
Create a secret based on a file, directory, or specified literal value.

A single secret may package one or more key/value pairs.

When creating a secret based on a file, the key will default to the basename of the file, and the value will default to
the file content. If the basename is an invalid key or you wish to chose your own, you may specify an alternate key.

When creating a secret based on a directory, each file whose basename is a valid key in the directory will be packaged
into the secret. Any directory entries except regular files are ignored (e.g. subdirectories, symlinks, devices, pipes,
etc).

Usage:
  oc create secret generic NAME [--type=string] [--from-file=[key=]source] [--from-literal=key1=value1] [--dry-run]
[options]

Examples:
  # Create a new secret named my-secret with keys for each file in folder bar
  oc create secret generic my-secret --from-file=path/to/bar
```

The command you reference is deprecated.

```
❯ oc secrets -h
Manage secrets in your project

Secrets are used to store confidential information that should not be contained inside of an image. They are commonly
used to hold things like keys for authentication to other internal systems like Docker registries.

Aliases:
secrets, secret

Usage:
  oc secrets [options]

Available Commands:
  add           DEPRECATED: secrets link
  link          Link secrets to a ServiceAccount
  new           DEPRECATED: create secret
```